lib/use_case/update_board.rb

```diff
@@ -10,7 +10,8 @@ def execute(player, at_index:)
     raise RangeError unless index_in_range?(at_index, board.length)
     raise DuplicationError unless board[at_index].nil?
 
-    @board_gateway.update(player, at_index)
+    board[at_index] = player
+    @board_gateway.update(board)
```

Comment:
I feel like it was more readable prior to the edit

Reply:
Yes, but then the actual update is made in the board gateway not in the update_board usecase